packages/kbn-reporting/export_types/csv/csv_searchsource_immediate.ts

```diff
@@ -108,9 +108,11 @@ export class CsvSearchSourceImmediateExportType extends ExportType<
     };
     const cancellationToken = new CancellationToken();
     const csvConfig = this.config.csv;
+    const taskInstanceFields = { startedAt: null, retryAt: null };
```

Comment:
Here is another example of why the "immediate" CSV export type doesn't fit well into the reporting framework: getting the export available to download immediately doesn't go through Task Manager.

Related: https://github.com/elastic/kibana/issues/164104